packages/paypal-commerce-utils/src/paypal-commerce-sdk.ts

```diff
@@ -42,6 +44,26 @@ export default class PayPalCommerceSdk {
         return this.window.paypalAxo;
     }
 
+    async getPayPalMessages(
+        paymentMethod: PaymentMethod<PayPalCommerceInitializationData>,
+        currencyCode: string,
+    ): Promise<PayPalMessagesSdk> {
+        if (!this.window.paypalMessages) {
+            const paypalSdkMessagesConfig = this.getPayPalSdkMessagesConfiguration(
+                paymentMethod,
+                currencyCode,
+            );
+
+            await this.loadPayPalSdk(paypalSdkMessagesConfig);
+        }
+
+        if (!this.window.paypalMessages) {
+            throw new PaymentMethodClientUnavailableError();
+        }
```

Comment:
We can optimize it by combining `if` checks. That way we can avoid redundant condition check.
 
```suggestion
      if (!this.window.paypalMessages) {
        const paypalSdkMessagesConfig = this.getPayPalSdkMessagesConfiguration(
            paymentMethod,
            currencyCode,
        );

      await this.loadPayPalSdk(paypalSdkMessagesConfig);
      
      if (!this.window.paypalMessages) {
          throw new PaymentMethodClientUnavailableError();
      }
  }
```

Reply:
We need to use the second `if` operator because for some reason the instance might be loaded with the wrong namespace and we need to check it

Reply:
Yes, we don't remove the second `if`, we just move it inside the first `if`, so after that we can avoid redundant condition check.

Reply:
Misunderstood a little bit. It makes sense. I did that for the `getPayPalAxo` function as well. Thanks!